css-layout-classes/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href="styles.css" rel="stylesheet">
+    <title>CSS-Layout-Classes</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-header">
+          <h1>Choose your character!</h1>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-one-third">
+          <img src="images/link.png" alt="ness">
+        </div>
+        <div class="column-one-third">
+          <h2>Super Smash Bros.<br>Ultimate.</h2>
+          <p>Super Smash Bros. Ultimate<br>received universal acclaim from bothcritics and players, with some critics
+            calling it the best installment in theseries. It received praise for its large
+            amount of content and fine-turning of existing Smash gameplay elements, although
+            its online mode was widely<br>criticized.</p>
+        </div>
+        <div class="column-one-third">
+          <img src="images/yoshi.png" alt="yoshi">
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-one-fourth">
+          <img src="images/kirby.png" alt="kirby">
+        </div>
+        <div class="column-one-fourth">
+          <img src="images/fox.png" alt="fox">
+        </div>
+        <div class="column-one-fourth">
+          <img src="images/pikachu.png" alt="pikachu">
+        </div>
+        <div class="column-one-fourth">
+          <img src="images/ness.png" alt="ness">
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-two-third">
```

Comment:
for the final two rows, there should be two columns, one that is 1/3 width for the image, and one that is 2/3 width for the text. Make sure to properly create those classes and explicitly assign them a width value.